### ISSUE TITLE
Winds app being detected as malware on BitDefender

### ISSUE THREAD

**Issue by antojk** [GetStream/Winds#596]
Bitdefender is classifying the App as a "malware" threat and quarantined the following executables:

- winds\winds.exe
- winds\resources\elevate.exe
- winds-updater\installer.exe

Any probable cause?

**Comment by tschellenbach** [comment]
That's weird, does bitdefender provide a reason?, perhaps ask them. Or use
the web version if you are concerned about this.

**Comment by tbarbugli** [comment]
TL;DR BitDefender is wrong about this.

Unfortunately BitDefender is giving false positives about Winds. We spent a great deal making sure that this was not a false positive. I did scan Winds.exe with several other AV software and they all came back clean. I reached out to BitDefender already to report this issue but they did not bother replying or take any action.

<rant>
If a data point was needed about how useless AV software is for PC/Mac/Linux pro-users, there you have it. My personal advice: if you know how to use a computer with common sense, remove BitDefender and enjoy the extra speed and beautiful RSS reader Winds.
</rant>

I am going to report this again to BitDefender and post this somewhere else; I think only some media pressure helps in this cases.
